kiosk/src/Models/Kiosk.ts

```diff
@@ -16,13 +16,16 @@ export class Kiosk {
     state: KioskState = KioskState.MainMenu;
 
     private readonly highScoresLocalStorageKey: string = "HighScores";
+    private readonly addedGamesLocalStorageKey: string = "UserAddedGames";
     private initializePromise: any;
     private siteElements: ChildNode[] = [];
     private intervalId: any;
     private readonly allScoresStateKey: string = "S/all-scores";
     private lockedGameId?: string;
     private launchedGame: string = "";
     private builtGamesCache: { [gameId: string]: BuiltSimJSInfo } = { };
+    private addedGameDescription: string = "Made with love in MakeCode Arcade";
```

Comment:
the descriptions can be fetched with `https://makecode.com/api/{shareId}` but this is already a big pr so we can do that later